h/views/api_auth.py

```diff
@@ -31,6 +32,43 @@ def __init__(self, context, request):
     @view_config(request_method='GET',
                  renderer='h:templates/oauth/authorize.html.jinja2')
```

Comment:
I can't find any description of what should happen when the request includes an invalid response mode in any spec, so I would err on the side of the decisions that oauthlib made which is a fallback to the `query` response mode.